code/obj/item/mob_parts.dm

```diff
@@ -252,7 +255,10 @@ ABSTRACT_TYPE(/obj/item/parts)
 			direction = turn(direction,180)
 
 		if (isitem(object))
-			object.streak_object(direction, src.streak_decal)
+			if (messy)
+				object.streak_object(direction, src.streak_decal)
+			else
+				object.streak_object(direction, null)
```

Comment:
why bother even streaking if its not messy

Reply:
dont get this

Reply:
I was using it because streaking doesn't throw the limbs, and skele limbs are boomerangs. I realized you can just turn that off with a var though, so I've reverted back to using ThrowRandom.